compiler/rustc_infer/src/infer/type_variable.rs

```diff
@@ -346,56 +344,6 @@ impl<'tcx> TypeVariableTable<'_, 'tcx> {
         )
     }
 
-    /// Finds the set of type variables that existed *before* `s`
-    /// but which have only been unified since `s` started, and
-    /// return the types with which they were unified. So if we had
-    /// a type variable `V0`, then we started the snapshot, then we
-    /// created a type variable `V1`, unified `V0` with `T0`, and
-    /// unified `V1` with `T1`, this function would return `{T0}`.
-    pub fn types_escaping_snapshot(&mut self, s: &super::Snapshot<'tcx>) -> Vec<Ty<'tcx>> {
```

Comment:
cc @nikomatsakis This one (and the `Instantiate` one above) is a bit strange, I wonder what happened here.

Reply:
Not sure. I'm not too bothered, to be honest, I'd be inclined to accept the PR, but I guess if we're worried we could dig into the history and see who the caller was and what's become of them.